CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -1,3 +1,13 @@
+# v1.8.2 - Changelog since v1.8.1
+
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Update to go 1.19.4 ([#1103](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1103), [@msau42](https://github.com/msau42))
```

Comment:
Is this generated by the script? Seems like the change log does not include the cherry-picks prs: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1115 is this expected for this repo's change-log?

Reply:
all of the PRs cherry picked by #1115 does not have release notes, so it's normal that the changelog does not include it.